### DISCUSSION TITLE
Spotify provider

### DISCUSSION THREAD

**Discussion by ghost** [TopiSenpai/LavaSrc#78] [General]
Good evening,
I would like to know how to set spotify as a provider.

**Comment by topi314** [comment]
spotify can't be a provider as it consumes the providers. that would result in an infinite loop

**Reply by ghost** [reply]
Okay thanks.

**Comment by ghost** [comment]
Last question, when i'm trying to play deezer music i got this error.

"Mp3 frame not found"

**Reply by topi314** [reply]
that suggests that your master decryption key is wrong, and no i can't help you with it

**Comment by ghost** [comment]
Where i can find a correct master decryption key ?

**Comment by ghost** [comment]
I just want the link
